tests/regrtest_data/pyi/a_module_that_we_definitely_dont_use_in_the_functional_tests.pyi

```diff
@@ -0,0 +1,5 @@
+# This module is named in a particular way to prevent test pollution by resolving all mentions of
+# foo to this stub file.
```

Comment:
```suggestion
# This module is named in a particular way to prevent test pollution. It was previously named 'foo' and
# all mentions of 'foo' were wrongly resolved to this stub file.
```